src/endpoints/models/DateModel.js

```diff
@@ -0,0 +1,86 @@
+export default class DateModel {
+  constructor ({ startDate, endDate, allDay }) {
+    this._startDate = startDate
+    this._endDate = endDate
+    this._allDay = allDay
+  }
+
+  // (!) startDate and endDate are not in German, but UTC TimeZone. They are sent in German timezone by Server, but without any
+  // timezone declaration, so we don't know if it's DaylightSavingTime or not. So we parse it in UTC and display it in UTC.
```

Comment:
We need some proper backend first :P

Reply:
Also now the time is displayed just as we get it from the server (indepent of the local time of the device). We have no other choice, and it's also at least good enough, since the user generally knows that the events are in germany and therefore in german (backend) timezone.